.rubocop.yml

```diff
@@ -30,3 +30,6 @@ Metrics/ClassLength:
 Performance/FixedSize:
   Exclude:
     - 'test/**/*'
+
+Style/AndOr:
+  EnforcedStyle: conditionals
```

Comment:
I'm ok with setting it to `always`. Mind if I open a separate PR to do so though? After working on refactoring some of the instances where we use `and` and `or` as control flow operators, it's going to take more than a few hours of work and double size of this PR. I can also add a comment to this line (e.g. `# TODO: remove this configuration and thereby use the default value of always`).